src/checkstyle/custom-checks.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+  "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+  "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
+  <module name="RegexpHeader">
+    <property name="fileExtensions" value="java"/>
+    <property name="headerFile" value="${checkstyle.header.file}"/>
+  </module>
+  <module name="TreeWalker">
+    <module name="RequireThis">
```

Comment:
Why do you want to enable `RequireThis`?

Reply:
Oops, done removed.